src/Cameras/babylon.cameraInputsManager.ts

```diff
@@ -28,6 +28,10 @@ module BABYLON {
             this.checkInputs = () => { };
         }
 
+		/**
+		 * Add an input method to a camera
+		 * @param input

```

Comment:
not sure "add" and "remove" need documentation (kind'a like a getter and a setter), BUT - if you already added it, please document the parameter as well.